tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/AgentConnectivityCheckTests.cs

```diff
@@ -43,17 +46,37 @@ public async Task DetectAgentUrl((string, string)[] environmentVariables)
 
             processInfo.Should().NotBeNull();
 
-            var result = await AgentConnectivityCheck.Run(processInfo);
+            _ = await AgentConnectivityCheck.Run(processInfo!);
 
             console.Output.Should().Contain(DetectedAgentUrlFormat("http://fakeurl:7777/"));
         }
 
+        [Theory]
+#if NETCOREAPP3_1_OR_GREATER
+        [InlineData("unix://file", "domain sockets")]
+#endif
+        [InlineData("http://127.0.0.1:8726/", "HTTP")]
```

Comment:
Worth adding a couple of tests with malformed URLs etc? e.g. `unix://`, `http://unix://127.0.0.1:8726`, `http://127.0` etc

Reply:
I'm not sure what you expect. It will fail, so what am I validating? 😛

Reply:
That you don't crash 😛